fuzzers/main.py

```diff
@@ -437,6 +437,7 @@ def start(self, corpus: Corpus) -> None:
                 str(corpus.corpus_for(self.target)),
                 str(corpus.artifacts_for(self.target)),
                 f'-artifact_prefix={corpus.artifacts_for(self.target)}/',
+                '-timeout=8000', # nayduck can sigstop the fuzzing process for ~2 hours at most
```

Comment:
Basically, if one fuzzer runner run (so one iteration of the loop) takes more than this timeout time in wall clock time, then it is considered as having entered an infinite loop and thus as a fuzzing crash.

However, as we're having sigstop/sigcont for the duration of a nayduck run, the wall clock time will run forward during this time. So what happened the first day was that two spurious timeouts were detected during the nayduck worker's test run.

With this updated timeout, infinite loops will take (much) longer (than the default 1200s) to detect, but it's not that bad (because the worst thing that could happen for us is we'd miss the infinite loop run if it were started between 1200 and 8000s before the restart, which happens daily; it doesn't actually increase the average time of a loop iteration) and avoids false positives.

Reply:
> Basically, if one fuzzer runner run (so one iteration of the loop) takes more than this timeout time in wall clock time, then it is considered as having entered an infinite loop and thus as a fuzzing crash.

Got it. Perhaps add a short comment to make it clear for others?